### PR TITLE
Only start autocomplete when next character is empty or a space.

### PR DESCRIPTION
**Pull request by tuga3d**
Just for discussion, do not commit.

*bugsquad node:* This is intended to solve #10695.

### CODE REVIEW

File: editor/code_editor.cpp [hunk 1]
```diff
@@ -1020,8 +1020,13 @@ void CodeTextEditor::_line_col_changed() {
 
 void CodeTextEditor::_text_changed() {
 
-	code_complete_timer->start();
-	idle->start();
+	String next_char;
+	next_char = text_editor->get_line(text_editor->cursor_get_line()).substr(text_editor->cursor_get_column(), 1);
```
Comment (akien-mga) [inline editor/code_editor.cpp]: No need to use two statements for this, you can do `String next_char = text_editor->...` directly.

Comment (tuga3d) [reply editor/code_editor.cpp]: thanks, i'll update. This is just a little hack to demonstrate how, i think, the behaviour of the popup should be. I don't know c++ to do a proper change. I've update the [bug(behaviour)](https://github.com/godotengine/godot/issues/10695) report with some animated gifs, the previous one was unclear.

**Comment by tuga3d** [comment]
well, i was doing my godoty things, and i thought i'll activate that fancy autocomplete braces to spare me some keystrokes, and now the next character is a space rule makes no sense.

**Comment by tuga3d** [comment]
closing the logic doesn't work.

**Pull request closed without merging**